eva_submission/eload_brokering.py

```diff
@@ -144,10 +145,26 @@ def _get_valid_vcf_files(self):
             valid_vcf_files.extend(files) if files else None
         return valid_vcf_files
 
+    def _generate_csv_mappings(self):
+        vcf_files_mapping_csv = os.path.join(self.eload_dir, 'vcf_files_mapping.csv')
```

Comment:
Can we re-use the same csv file created in the [validation](https://github.com/EBIvariation/eva-submission/blob/cb2866a047aca4b589bcca14f3a311436f0eb07f/eva_submission/eload_validation.py#L231) for this?  If not, we should perhaps name the file `vcf_files_to_broker.csv` so we don't overwrite the other one.

Reply:
They are different so I change the names.